### DISCUSSION TITLE
Start the deployment server

### DISCUSSION THREAD

**Discussion by mohammedjameel24** [Praful932/Kitabe#127] [General]
I cannot create a virtual env , if created i cannot start it using the commands given in project contribute.md and while installing the requirements it throws errors for some deprecated 📦

**Comment by maneprajakta** [comment]
what is your os?
Also python version?
attach ss of error you are getting.

**Comment by mohammedjameel24** [comment]
Windows 
Python 3.9

**Comment by maneprajakta** [comment]
As mentioned in Contribute.md the prjoect works on python version 3.8
`The Project works seamlessly on Python version 3.8.6`

**Comment by mohammedjameel24** [comment]
How do i downgrade?

**Reply by Praful932** [reply]
You can use [two python versions](https://www.youtube.com/watch?v=OdIHeg4jj2c) too,but that would make things a bit complex.
Most of the packages are still not fully supported for Python 3.9, you can download the 3.8 version from the [offical website](https://www.python.org/downloads/windows/)
